pkg/scaffold/constants.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scaffold
+
+import (
+	"path/filepath"
+)
+
+const (
+	// file modes
+	defaultDirFileMode  = 0750
+	defaultFileMode     = 0644
+	defaultExecFileMode = 0744
+
+	noOverwriteTrue  = true
+	noOverwriteFalse = false
+
+	mustNotExistTrue  = true
+	mustNotExistFalse = false
+
+	// Separator to statically create directories.
+	filePathSep = string(filepath.Separator)
+
+	// Generated project structure:
```

Comment:
let's get rid of the file structure comments from line 36 to 51. it is documented in the `operator-sdk new` command already.